api/src/components/publication/controller.ts

```diff
@@ -67,6 +67,28 @@ export const get = async (
     }
 };
 
+export const getPubSeed = async (
+    event: I.APIRequest<undefined, I.GetPublicationQueryParams>
+): Promise<I.JSONResponse> => {
+    try {
+
+        const publication = await publicationService.getPubSeed(event.queryStringParameters.title);
+       
+        if(publication !== null) {
+            return response.json(200, {
+                publication
+            })
+        }
+        return response.json(404, {
```

Comment:
I don't a 404 is suitable here. Because it's not like fetching /publications/{publicationId} where you're trying to get a specific publication.
You can either change the service from "findMany" to "findFirst" and then you can keep this 404 as it is. Either simply return whatever the service is returning => array with pubs or empty array